ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -86,7 +75,7 @@ import Purchases
     }
 
     @objc public static func invalidatePurchaserInfoCache() {
-        Purchases.shared.invalidatePurchaserInfoCache()
+        Purchases.shared.invalidateCustomerInfoCache()
```

Comment:
note that I'm keeping the public API the same, just updating internal calls. Public API updates can be in their own PR. 
They can even wait until somewhere in the future, and we could release hybrids with this